src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/PartiesNotifiedMapping.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.partiesnotified.PartiesNotifiedRequestPayload;
+import uk.gov.hmcts.reform.sscs.model.partiesnotified.ServiceData;
+
+import static uk.gov.hmcts.reform.sscs.helper.service.HearingsServiceHelper.getVersion;
+
+public final class PartiesNotifiedMapping {
+    private PartiesNotifiedMapping() {
```

Comment:
@NoArgsConstructor(access = AccessLevel.PRIVATE) just a suggestion.

Reply:
Doesn't work unfortuantely, tried it before, pmd will flag it as a helper without a private constructor :(

Reply:
Strange, alright no worries